picard/coverart/image.py

```diff
@@ -77,6 +78,7 @@ def __init__(self, data, prefix='picard', suffix=''):
                 with os.fdopen(fd, "wb") as imagefile:
                     imagefile.write(data)
                 _datafiles[self._hash] = self._filename
+                periodictouch.register_file(self._filename)
```

Comment:
I think those functions should be no-op on anything but macos.

Reply:
I didn't want to push that logic all over the place and keep the decision of whether to start this at a single place to tagger.py (which basically controls bootstrapping the application, and hence I think there this decision belongs to).

But I changed the module to not fill the temp file set if the timer is not active, see https://github.com/metabrainz/picard/pull/2099/commits/7e3e8954cb4ea3dc201500231699c9e7e7e63569

Reply:
Yes, it is even better. Nice work.